docker/geoserver/tasks.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#########################################################################
+#
+# Copyright (C) 2023 OSGeo
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+#########################################################################
+import os
+import os
+import logging
+import time
+import base64
+import requests
+from pathlib import Path
+from invoke import task
+
+logger = logging.getLogger(__name__)
+
+
+@task
+def configure_geoserver(ctx):
+    _configure_geoserver_password()
+    _initialized(ctx)
+
+
+def _configure_geoserver_password():
+    print(
+        "************************configuring Geoserver credentials*****************************"
+    )
+    GEOSERVER_LB_PORT = os.getenv("GEOSERVER_LB_PORT", 8080)
+    GEOSERVER_ADMIN_USER = os.getenv("GEOSERVER_ADMIN_USER", "admin")
+    GEOSERVER_ADMIN_PASSWORD = os.getenv("GEOSERVER_ADMIN_PASSWORD", "geoserver")
+    GEOSERVER_FACTORY_PASSWORD = os.getenv("GEOSERVER_FACTORY_PASSWORD", "geoserver")
+    geoserver_rest_baseurl = f"http://localhost:{GEOSERVER_LB_PORT}/geoserver/rest"
```

Comment:
a strayed localhost, I suspect?!

Reply:
Nope it's ok. It will always refer to the local container.